quipucords/tests/api/scantask/test_scantask_get_facts.py

```diff
@@ -1,56 +1,47 @@
 """Test ScanTask.get_facts."""
 
+from functools import partial
+
 import pytest
 
-from api.models import (
-    RawFact,
-    ScanJob,
-    ScanTask,
-)
-from tests.factories import InspectGroupFactory, InspectResultFactory
-from tests.utils.facts import random_name, random_value
+from api.models import ScanTask
+from tests.factories import ScanTaskFactory
 
 
 @pytest.fixture
-def raw_facts_list():
+def raw_facts_list(faker):
     """Fixture representing a list of random facts."""
-    systems = []
-    for _ in range(50):
-        systems.append({random_name(): random_value() for _ in range(100)})
-    return systems
+    return [
+        {
+            "name": letter,
+            "boolean": faker.pybool(),
+            "int": faker.pyint(max_value=100),
+            "str": faker.slug(),
```

Comment:
I would suggest giving the keys random `faker.slug()` values just to make it very obvious for future debugging developers that these are purely synthetic values, never to be confused with the `int` or `str` types.

Reply:
ok!

Reply:
updated =D

Reply:
Did I miss a change? I still see the hard-coded keys that look like types. See also https://github.com/quipucords/quipucords/pull/2600#discussion_r1569448915

Reply:
sorry, I misunderstood what you were proposing. see https://github.com/quipucords/quipucords/pull/2600#discussion_r1569503580